packages/contracts-bedrock/scripts/DeployPeriphery.s.sol

```diff
@@ -468,7 +468,7 @@ contract DeployPeriphery is Deployer {
     /// @notice installs the OnChain AuthModule on the Faucet contract.
     function installOnChainAuthModule() public broadcast {
         string memory moduleName = "OnChainAuthModule";
-        Faucet faucet = Faucet(mustGetAddress("Faucet"));
+        Faucet faucet = Faucet(mustGetAddress("FaucetProxy"));
```

Comment:
While testing deploy realized that `configure` should be called on the Proxy contract and not the implementation contract.